src/App.tsx

```diff
@@ -53,18 +54,42 @@ export const App = () => {
     window.close();
   }
 
+  const initialiseRouteTranscript = () => {
```

Comment:
nit: i know we're from the commonwealth, but US english is probably preferred :laughing:

Reply:
Changed to `init`